test/Microsoft.AspNetCore.CookiePolicy.Test/CookieConsentTests.cs

```diff
@@ -626,7 +632,13 @@ public class CookieConsentTests
             Assert.Equal(Net.Http.Headers.SameSiteMode.Strict, consentCookie.SameSite);
             Assert.NotNull(consentCookie.Expires);
 
-            Assert.Equal(httpContext.Response.Headers[HeaderNames.SetCookie], httpContext.Response.Headers["ManualCookie"]);
+            cookies = SetCookieHeaderValue.ParseList(httpContext.Response.Headers["ManualCookie"]);
+            Assert.Equal(1, cookies.Count);
+            var manualCookie = cookies[0];
+            Assert.Equal(consentCookie.Name, manualCookie.Name);
+            Assert.Equal(consentCookie.Value, manualCookie.Value);
+            Assert.Equal(consentCookie.SameSite, manualCookie.SameSite);
+            Assert.NotNull(manualCookie.Expires); // Expires may not exactly match to the second.
```

Comment:
Nit: do you care about doing a rough match at least?  Parse and check for same day or something?

Reply:
The entire field is unstable as an added second could roll any or all fields if you run it on the minute, hour, day, month, or year boundary. Admittedly each stage gets less likely.